charts/posthog/values.yaml

```
@@ -21,8 +21,8 @@ sentryDSN:
 clickhouseOperator:
   # -- Whether to install clickhouse. If false, `clickhouse.host` must be set
   enabled: true
-  # -- Which namespace to install clickhouse operator to
-  namespace: "posthog"
+  # -- Which namespace to install clickhouse operator to (defaults to namespace chart is installed to)
+  namespace: "" 
```

If it's empty str, then default function thinks that's a reasonable value only for no value will default be used and that's what we want.

```suggestion
  namespace:
```